.env

```diff
@@ -0,0 +1 @@
+VITE_VERSION=0.5
```

Comment:
Should this file be committed? Is it an example env? I guess the CI should be setting the prod version.

Reply:
I don't understand why this file came back. It was deleted but then something funky happened ... when you look at the branch graph it does not look right ...